packages/playwright-core/src/client/page.ts

```diff
@@ -560,17 +559,9 @@ export class Page extends ChannelOwner<channels.PageChannel> implements api.Page
     await this.close();
   }
 
-  private async _waitForActiveRouteHandlersToFinish() {
-    const promises = this._routes.map(routeHandler => routeHandler.stopAndWaitForRunningHandlers(this));
-    promises.push(...this._browserContext._routes.map(routeHandler => routeHandler.stopAndWaitForRunningHandlers(this)));
```

Comment:
Since we never call `stopAndWaitForRunningHandlers` on context's handlers and pass `page` anymore, perhaps we do not need the parameter and the map from page to activations?

Reply:
Good point. Changed it to  `RouteHandler._ignoreException`.